Firmware/ESP32/Hardware_bringup/lib/DEV/dev_battery.c

```diff
@@ -9,16 +9,19 @@
 
 #include "dev_battery.h"
 
-// TableUV Lib
-#include "../IO/io_ping_map.h"
-
-// External Lib
-#include "driver/gpio.h"
-
 /////////////////////////////////
 ///////   DEFINITION     ////////
 /////////////////////////////////
-
+typedef enum {
+    CHARGING,                   // STAT Pin Low
+    CHARGE_COMPLETE_SLEEP,      // STAT Pin HIGH
+    CHARGER_FAULT               // STAT Pin Blinkings
+} charger_ic_status_t;
```

Comment:
Minor: Prefer '_E' to present enum, '_S' to present Struct, and '_t' only for typedef of pointer and name replacement

Reply:
Bring enum to .h